modin/numpy/math.py

```diff
@@ -44,6 +46,7 @@ def absolute(
 def add(
     x1, x2, out=None, where=True, casting="same_kind", order="K", dtype=None, subok=True
 ):
+    x1 = try_convert_from_interoperable_type(x1)
```

Comment:
There are many functions that are not tested: `add`, `divide`, `float_power`, `floor_divide`, `power`, `prod`, `multiply`, `remainder`, `subtract`, `true_divide`, `mean`, `maximum`, `minimum`, `amax`, `amin`. What is the plan for adding tests?

Reply:
We test the `array.add`, etc, which these call, so I think that should be ok?

Reply:
So is this a mistake in `codecov`?

Reply:
Yeah - I think codecov is complaining that `modin.numpy.add` isn't being tested (which is technically true), but `modin.numpy.add` is literally just a wrapper for `modin.numpy.array.add`, which is tested, so I think its fine not to test the wrapper?

Reply:
Ok! Thank you for the clarification.